apps/etterlatte-behandling/src/main/kotlin/sak/SakService.kt

```diff
@@ -74,4 +97,34 @@ class RealSakService(private val dao: SakDao) : SakService {
     override fun finnSak(id: Long): Sak? {
         return dao.hentSak(id)
     }
+
+    private fun finnEnhetForTemaOgOmraade(tema: String, omraade: String): ArbeidsFordelingEnhet {
+        val enheter = norg2Klient.hentEnheterForOmraade(tema, omraade)
+
+        return when {
+            enheter.isEmpty() -> throw IngenEnhetFunnetException(omraade, tema).also { logger.warn(it.message) }
+            else -> enheter.first()
```

Comment:
Jeg har ikke sett noe på det, så det stemmer sikkert, men kan vi bare anta at vi vil ha det første treffet?

Reply:
Vi har ingen mulighet å bedømme hvilken hvis det er mer enn en.

Valgene er vel:

* First
* First but log that there were more than one
* Throw and log that there were more than one

Reply:
Vi var vel så vidt inne på det før, kanskje, men vet ikke om det hadde gitt mening å sjekket på at den første isåfall er en av de gyldige enhetene vi har definert: 

![image_480](https://user-images.githubusercontent.com/1083866/229086217-a4740661-8273-484b-adaf-093db274929c.png)

Eller garanteres det av tjenesten kanskje? Så lenge vi får en av de som er definert over, så tenker jeg det er helt greit.

Reply:
Config i norg for EYO og EYB skal være samme som PEN - og PEN har fungert fint i pesys ?

Reply:
Ok, da er det sikkert fint 👍